src/Sulu/Component/HttpCache/Tests/Unit/Handler/UrlHandlerTest.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) MASSIVE ART WebServices GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Component\HttpCache\Tests\Unit\Handler;
+
+use FOS\HttpCache\ProxyClient\Invalidation\PurgeInterface;
+use FOS\HttpCache\ProxyClient\ProxyClientInterface;
+use Prophecy\Argument;
+use Sulu\Component\HttpCache\Handler\UrlHandler;
+use Sulu\Component\HttpCache\HandlerInterface;
+use Sulu\Component\Webspace\Url\ReplacerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\RequestStack;
+
+class UrlHandlerTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @var HandlerInterface
+     */
+    private $handler;
+
+    /**
+     * @var ProxyClientInterface
+     */
+    private $proxyClient;
+
+    /**
+     * @var RequestStack
+     */
+    private $requestStack;
+
+    /**
+     * @var Request
+     */
+    private $request;
+
+    /**
+     * @var ReplacerInterface
+     */
+    private $replacer;
+
+    /**
+     * @var string
+     */
+    private $host = 'sulu.io';
+
+    public function setUp()
+    {
+        parent::setUp();
```

Comment:
This call is not necessary, because the parent `setUp` method is implemented empty.